<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>801</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="del_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>610</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>740</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>总数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DIR_Anno">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>704</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>\\WEINAIKE\FFOutput\AAA\JPEGImages</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>340</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NUM">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>740</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>680</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>类别目录地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>310</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="del_5">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>610</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="del_1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="name4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>640</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="upper">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>740</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上一批</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>640</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="name2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="del_6">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>610</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>340</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>707</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>标注文件地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DIR">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>680</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>\\WEINAIKE\FFOutput\AAA\classes-mid-bak\包心菜</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>740</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>下一批</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>280</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>740</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>起始图片编号：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>740</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_3">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>280</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ALL">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>740</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="name6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>640</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>label_8</zorder>
    <zorder>lineEdit_DIR_Anno</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>lineEdit_NUM</zorder>
    <zorder>label</zorder>
    <zorder>name3</zorder>
    <zorder>del_5</zorder>
    <zorder>label_3</zorder>
    <zorder>label_2</zorder>
    <zorder>name4</zorder>
    <zorder>upper</zorder>
    <zorder>name5</zorder>
    <zorder>name2</zorder>
    <zorder>name1</zorder>
    <zorder>label_6</zorder>
    <zorder>label_9</zorder>
    <zorder>label_1</zorder>
    <zorder>lineEdit_DIR</zorder>
    <zorder>next</zorder>
    <zorder>del_2</zorder>
    <zorder>label_7</zorder>
    <zorder>load</zorder>
    <zorder>del_3</zorder>
    <zorder>lineEdit_ALL</zorder>
    <zorder>name6</zorder>
    <zorder>del_1</zorder>
    <zorder>del_4</zorder>
    <zorder>del_6</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label_fullImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>960</width>
       <height>540</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>FULL IMAGE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_part">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>540</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Part</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>640</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图片编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ALL_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>640</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="load_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>640</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NUM_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>640</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="upper_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>640</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上一张</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DIR_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>570</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>\\WEINAIKE\FFOutput\AAA\classes-mid-bak\黄瓜</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>571</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>类别内容目录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>640</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>总数：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next_2">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>640</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>下一张</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name1_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>740</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>603</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>标注文件地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DIR_Anno_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>600</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>\\WEINAIKE\FFOutput\AAA\JPEGImages</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_rect">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>680</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示标注框</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>680</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示标签</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 3</string>
    </attribute>
    <widget class="QLabel" name="label_fullImage_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>960</width>
       <height>540</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>FULL IMAGE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>593</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>标注文件地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>561</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>标注图片地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DIR_Anno_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>590</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:\ImageSet\ImageNet\val</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DIR_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>560</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:\ImageSet\ImageNet\JPEG</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upper_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>630</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>上一张</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ALL_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>630</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>630</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>总数：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>630</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图片编号：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next_3">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>630</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>下一张</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NUM_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>630</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_rect_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>670</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示标注框</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>670</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>显示标签</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name1_3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>550</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
